ecms_base/ecms_base.install

```diff
@@ -66,4 +66,9 @@ function ecms_base_install() {
       ->addContentTypeToWorkflow($type->id());
   }
 
+  // Delete the default 'contact' webform.
+  \Drupal::entityTypeManager()
```

Comment:
Does this remove the 'Contact' node that is created too? Should it?

Reply:
Yes, good call. I've added an extra step to the install hook.